browser-test/src/admin_predicates.test.ts

```diff
@@ -316,6 +332,8 @@ describe('create and edit predicates', () => {
     await applicantQuestions.clickNext()
     await applicantQuestions.answerNumberQuestion('123')
     await applicantQuestions.clickNext()
+    await applicantQuestions.answerCurrencyQuestion('100.02')
```

Comment:
FYI: I have a follow-on PR to make this test do negative evaluations too.  Currently the server just has to evaluate true to everything and this passes.